### ISSUE TITLE
Need some kind of loading indicator

### ISSUE THREAD

**Issue by foolip** [GoogleChrome/confluence#136]
The API Catalog view with the default selection of browsers took a few seconds to load after clicking "Expand All". With more browsers one would start to wonder what's happening, so a spinner of some sorts to indicate network activity would be good. And stop it if the fetch fails :)

**Comment by mdittmer** [comment]
I don't think this issue is related to clicking "Expand All", but rather, just the load time. Right now we have a (mostly correct) progress bar at the top of the table view, but it's rather subtle. We can add a toast after a timeout (at least, for now) and further investigate how to optimize first load time.

**Comment by foolip** [comment]
Yep, I was confused by the fact that I've always clicked "Expand All" without waiting for the load.

As discussed on chat, what we should have is probably a toast that shows up after 3 seconds of waiting that says "Still working, see issue bla for why it can be slow"
